src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
ok, seems i have to maintain a fork. i like this library, but need to handle 5.3 version.